stock_quant_strap_product/views/product_template_views.xml

```diff
@@ -13,4 +13,4 @@
             </xpath>
         </field>
     </record>
-</odoo>
+</odoo>
```

Comment:
Still there is changes in the file...
@eHanseFrank  Can you try to avoid using `git add .`?

Reply:
understood.